internal/pkg/cli/app_package.go

```diff
@@ -40,11 +49,54 @@ func NewPackageAppOpts() *PackageAppOpts {
 
 // Ask prompts the user for any missing required fields.
 func (opts *PackageAppOpts) Ask() error {
+	if opts.AppName == "" {
+		names, err := opts.listAppNames()
+		if err != nil {
+			return err
+		}
+		if len(names) == 0 {
+			return errors.New("there are no applications in the workspace, run `archer init` first")
+		}
+		app, err := opts.prompt.SelectOne("Which application's CloudFormation template would you like to generate?", "", names)
+		if err != nil {
+			return fmt.Errorf("prompt application name: %w", err)
+		}
+		opts.AppName = app
+	}
+	if opts.EnvName == "" {
+		names, err := opts.listEnvNames()
+		if err != nil {
+			return err
+		}
+		env, err := opts.prompt.SelectOne("Which environment's configuration would you like to use for your stack's parameters?", "", names)
```

Comment:
This prompt also a bit weirdly worded. What's an environment configuration?

Reply:
Ah yeah, I meant if they want to use environment overrides from their manifest we need to know their environment's name.

Changed to `"Which environment would you like to create this stack for?"`